gtsfm/multi_view_optimizer.py

```diff
@@ -24,6 +24,8 @@
 from gtsfm.bundle.bundle_adjustment import BundleAdjustmentOptimizer
 from gtsfm.data_association.data_assoc import DataAssociation
 
+#used to save result_metrics files within React application for internal rendering
+RESULT_METRICS_REACT_PATH = Path(__file__).resolve().parent.parent / "rtf_vis_tool" / "src" / "result_metrics"
```

Comment:
I may have missed it, but are we creating two copies of the files? Or are we creating symlinks?

Reply:
So the problem with directly importing the result_metric files in the gtsfm folder into my React application is that React will not let me import anything outside it's source code folder. Because of this limitation, the next best option would be to save the result_metrics files in twice. Once in the gtsfm folder like normal and once within my React project folder.